grails-app/conf/BuildConfig.groovy

```diff
@@ -146,6 +144,20 @@ grails.project.dependency.resolution = {
 
         // for com.google.common
         compile 'com.google.guava:guava:12.0'
+
+        compile 'org.jxls:jxls:2.8.1' // TODO: This is the last version for java 7. After migration to Java 8 upgrade this to 2.9+
+        compile 'org.jxls:jxls-poi:1.0.9' // this one supports apache poi v3 (anything above is for v4, which is Java 8)
+
+        compile 'org.slf4j:slf4j-api:1.7.26'
+        compile 'org.slf4j:slf4j-log4j12:1.7.26'
+        compile 'org.slf4j:jul-to-slf4j:1.7.26'
+        compile 'org.slf4j:jcl-over-slf4j:1.7.26'
+
+        // Build to avoid pulling org.slf4j 1.5.8 on run-app
+        build 'org.slf4j:slf4j-api:1.7.26'
+        build 'org.slf4j:slf4j-log4j12:1.7.26'
+        build 'org.slf4j:jul-to-slf4j:1.7.26'
+        build 'org.slf4j:jcl-over-slf4j:1.7.26'
```

Comment:
I think this whole block can be removed. Grails' dependency management does the right thing when presented with the `"org.slf4j.*:1.7.33"` dependencies declared above